scripts/test_insta.sh

```
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# 설정 (init_data.sh와 동일하게)
+API_URL="http://localhost:8080"
+EMAIL="test1@example.com" # 기존 테스트용 이메일 유지
```

Shellcheck (0.11.0)

[warning] EMAIL appears unused. Verify use (or export if used externally).

(SC2034)